iotedgedev/version.py

```diff
@@ -0,0 +1,6 @@
+import sys
+
+PY35 = sys.version_info >= (3, 5)
```

Comment:
If this check is supposed to be consistent across `iotedgedev`, you could put it in `iotedgedev/__init__.py` so that it is evaluated upon package load.

Reply:
I tested this and it has no effect on the time it takes to throw an error. Furthermore, in neither scenario is the assert actually thrown, which is interesting because I got the desired output with the tests...

Reply:
I'm going to keep it as is, as it has no affect on performance or error-throw time.

Reply:
This comment wasn't aimed for performance - it was aimed to reduce code redundancy.

Reply:
And if the expected behavior  is not working, seems like it would be good to investigate and fix/refactor.

Reply:
Fair enough and good point. I'll find an appropriate solution.

Reply:
Confirmed that the version assert only provided value for testing. It has been removed.